ofrak_core/test_ofrak/components/test_free_space.py

```diff
@@ -19,11 +19,11 @@
 async def resource_under_test(ofrak_context: OFRAKContext) -> Resource:
     resource = await ofrak_context.create_root_resource(
         "mock_memory_region",
-        b"\xff" * 0x100,
+        b"\xff" * 0x110,
```

Comment:
Line 21 is confusing -- should this be renamed to "mock_program"? (I realize this is my fault :) ).